draft-ietf-ppm-dap.md

```diff
@@ -1813,12 +1813,11 @@ same `aggregation-job-id` but with a different `AggregationJobInitReq` in the
 body MUST fail with an HTTP client error status code. For further requests with
 the same `AggregationJobInitReq` in the body, the Helper SHOULD respond as it
 did for the original `AggregationJobInitReq`, or otherwise fail with an HTTP
-client error status code.
-
-Additionally, it is not possible to rewind or reset the state of an aggregation
-job. Once an aggregation job has been continued at least once (see
-{{agg-continue-flow}}), further requests to initialize that aggregation job MUST
-fail with an HTTP client error status code.
+client error status code. The Helper SHOULD allow the aggregation job to be
+restarted even if the aggregation job has been continued (see
+{{agg-continue-flow}}), to allow for a Leader to recover from state skew between
```

Comment:
Restarting entails re-sending the `AggregationJobInitReq` _unchanged_, correct?

Reply:
Yes, this is spelled out in the "Recovering from Aggregation Step Skew" section; I changed the wording to spell it out here, too.